README.md

```diff
@@ -19,7 +19,7 @@ But if you are interested, feel free to help test it.
 
 ```sh
 scoop install git python llvm cmake curl
-pip install mako
+python3 -m pip install mako
```

Comment:
What about suggesting to use a virtualenv while at it?

Reply:
Ideally, I don't want users to worry about the Python package. However, the recent update makes this a bit unfortunate.
I'll keep an eye on this. Maybe we should just mention mako is required and users are free to decided how to include it.